apps/desktop-wallet/src/api/apiDataHooks/wallet/useFetchWalletBalancesTokensByAddress.ts

```diff
@@ -25,14 +25,17 @@ const combineBalancesByAddress = (
   ...combineIsLoading(results)
 })
 
-const {
-  useData: useFetchWalletBalancesTokensByAddress,
-  DataContextProvider: UseFetchWalletBalancesTokensByAddressContextProvider
-} = createDataContext<AddressTokensBalancesQueryFnData, AddressesTokensBalances['data']>({
+const { useData, DataContextProvider } = createDataContext<
+  AddressTokensBalancesQueryFnData,
+  AddressesTokensBalances['data']
+>({
   useDataHook: useFetchWalletBalancesTokens,
   combineFn: combineBalancesByAddress,
   defaultValue: {}
 })
 
+const useFetchWalletBalancesTokensByAddress = useData
+const UseFetchWalletBalancesTokensByAddressContextProvider = DataContextProvider
```

Comment:
sorry man but this code is always so hard to understand every time I want to dive into the implementation of these hooks. This tiny change helps a little bit imo